test_project/store/src/main/java/test/sergeyvalyushko/store/Catalog.java

```diff
@@ -0,0 +1,64 @@
+package test.sergeyvalyushko.store;
+
+import org.xml.sax.SAXException;
+import test.sergeyvalyushko.common.ReflectionUser;
+import javax.xml.parsers.ParserConfigurationException;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+public class Catalog {
+
+    String location = "test.sergeyvalyushko.store";
+    ReflectionUser<Category> reflection = new ReflectionUser<Category>();
+
+    protected void displayCatalog() {
```

Comment:
think about refactoring three display* methods, cause they are very similar and i think you can extract some common logic to one method